From 2352b56f80c4d3924065c5bdeb8a0adaf4127894 Mon Sep 17 00:00:00 2001
From: Catmengi <138163939+catmengi@users.noreply.github.com>
Date: Tue, 19 Nov 2024 09:48:28 +0000
Subject: [PATCH 51/56] arch: arm64: boot: dts: qcom: sm8150.dtsi: change reset
 name and freq-table-hz of ufs_mem_hc

This is needed in some nabu tablets to fix UFS
---
 arch/arm64/boot/dts/qcom/sm8150.dtsi | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/sm8150.dtsi b/arch/arm64/boot/dts/qcom/sm8150.dtsi
index 62b2aedeeefd..e8f1de30f65d 100644
--- a/arch/arm64/boot/dts/qcom/sm8150.dtsi
+++ b/arch/arm64/boot/dts/qcom/sm8150.dtsi
@@ -2088,7 +2088,7 @@ ufs_mem_hc: ufshc@1d84000 {
 			lanes-per-direction = <2>;
 			#reset-cells = <1>;
 			resets = <&gcc GCC_UFS_PHY_BCR>;
-			reset-names = "rst";
+			reset-names = "core_reset";
 
 			iommus = <&apps_smmu 0x300 0>;
 
@@ -2117,11 +2117,11 @@ ufs_mem_hc: ufshc@1d84000 {
 				<0 0>,
 				<0 0>,
 				<37500000 300000000>,
+				<37500000 300000000>,
 				<0 0>,
 				<0 0>,
 				<0 0>,
-				<0 0>,
-				<0 300000000>;
+				<0 0>;
 
 			status = "disabled";
 		};
-- 
2.48.1

